connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/RelationalGroupedDataset.scala

```diff
@@ -225,3 +234,28 @@ class RelationalGroupedDataset protected[sql] (
     toDF(colNames.map(colName => functions.sum(colName)))
   }
 }
+
+private[sql] object RelationalGroupedDataset {
```

Comment:
For the record, I would also be fine with you using the proto enums. From the look of it there is no real way a user can interact with this.

Reply:
I see. Reuse proto now.